tests/test_quick.py

```diff
@@ -22,9 +22,47 @@ def test_run(mrcs_file, poses_file):
         mrcs_file,
         '-o', 'output',
         '--lr', '.0001',
+        '--num-epochs', '20',
         '--seed', '0',
         '--poses', poses_file,
         '--zdim', '10',
-        '--pe-type', 'gaussian'
+        '--pe-type', 'gaussian',
     ])
-    train_vae.main(args)
\ No newline at end of file
+    train_vae.main(args)
```

Comment:
The `test_quick.py` test script now does a lot more testing beyond `train_vae` - `analyze`/`analyze_landscape`/`graph_traversal`/`eval_vols`. It still runs fairly fast using the arguments supplied here. This actually allowed me to uncover a few more bugs in the `analyze_landscape` module, on which I can open up issues in the future.